src/FourierFlows.jl

```diff
@@ -159,11 +159,11 @@ function show(io::IO, params::AbstractParams)
   showstring = ""
   for name in names[1:end-1]
     field = getproperty(params, name)
-    showstring = string(showstring, "  ├───── parameter: " * string(name) * " -> ", summary(field), "\n")
+    showstring = string(showstring, "  ├───── parameter: " * string(name) * " -> ", typeof(field) <: Function ? "Function" : summary(field), "\n")
```

Comment:
Not sure what `summary` does but can't you use dispatch there rather than an if-statement here?

Reply:
Good point! e675de2 deals with this.